Applications/spire/source/charting/chart_view.cpp

```diff
@@ -12,6 +11,13 @@ using namespace boost::posix_time;
 using namespace Nexus;
 using namespace Spire;
 
+namespace {
+  QVariant chart_value_to_variant(ChartValue::Type type,
+      const ChartValue& value) {
+    return QVariant::fromValue(static_cast<Money>(value));
```

Comment:
Pass by value, furthermore implement this function for all possible ```ChartValue::Type```s.

Reply:
Why is the ChartValue passed by value here?

Reply:
ChartValue's should always be passed by value, it's an 8 byte numeric type. Same reason you always pass an int or a double by value.